libs/ledgerjs/packages/types-live/src/postOnboarding.ts

```diff
@@ -8,9 +8,14 @@ export enum PostOnboardingActionId {
   claimMock = "claimMock",
   migrateAssetsMock = "migrateAssetsMock",
   personalizeMock = "personalizeMock",
+  assetsTransferMock = "assetsTransferMock",
+  buyCryptoMock = "buyCryptoMock",
+  customImageMock = "customImageMock",
+  recoverMock = "recoverMock",
```

Comment:
I'm not sure it was necessary to add a mock for all those actions, the mocks are mainly here to test the logic of the post onboarding itself without having to go through the business specific flow of all the actions, and 3 or 4 mocked actions might be enough for that.

Reply:
I think also `personalizeMock` is referring to the same thing as `customImageMock`: custom lock screen.
Same for `migrateAssetsMock` and `assetsTransferMock`.